docs/cli-manual.md

```diff
@@ -0,0 +1,252 @@
+```
+
+If you see the version of tkctl tool and version of TiDB operator installed in target cluster or "No TiDB Controller Manager found, please install one first.", `tkctl` is correctly configured to access your cluster.
+
+# Commands
+
+## tkctl version
+
+This command used to show the version of **tkctl** and **tidb-operator** installed in target cluster.
+
+Example:
+```
+$ tkctl version
+Client Version: v1.0.0-beta.1-p2-93-g6598b4d3e75705-dirty
+TiDB Controller Manager Version: pingcap/tidb-operator:latest
+TiDB Scheduler Version: pingcap/tidb-operator:latest
+```
+
+## tkctl list
+
+This command used to list all tidb clusters installed.
+
+| Flags | Shorthand | Description |
+| ----- | --------- | ----------- |
+| --all-namespaces | -A | search all namespaces |
+| --output | -o | output format, one of [default,json,yaml], the default format is `default` |
+
+Example:
+
+```
+$ tkctl list -A
+NAMESPACE NAME           PD    TIKV   TIDB   AGE
+foo       demo-cluster   3/3   3/3    2/2    11m
+bar       demo-cluster   3/3   3/3    1/2    11m
+```
+
+## tkctl use
+
+This command used to specify the current TiDB cluster to use, the other commands could omit `--tidbcluster` option and defaults to select current TiDB cluster if there is a current TiDB cluster set.
+
+Example:
+
+```
+$ tkctl use --namespace=foo demo-cluster
+Tidb cluster switched to foo/demo-cluster
+```
+
+## tkctl info
+
+This command used to get the information of TiDB cluster, the current TiDB cluster will be used if exists.
+
+| Flags | Shorthand | Description |
+| ----- | --------- | ----------- |
+| --tidb-cluster | -t | select the tidb cluster, default to current TiDB cluster |
+
+Example:
+
+```
+$ tkctl info
+Name:               demo-cluster
+Namespace:          foo
+CreationTimestamp:  2019-04-17 17:33:41 +0800 CST
+Overview:
+         Phase    Ready  Desired  CPU    Memory  Storage  Version
+         -----    -----  -------  ---    ------  -------  -------
+  PD:    Normal   3      3        200m   1Gi     1Gi      pingcap/pd:v2.1.4
+  TiKV:  Normal   3      3        1000m  2Gi     10Gi     pingcap/tikv:v2.1.4
+  TiDB   Upgrade  1      2        500m   1Gi              pingcap/tidb:v2.1.4
+Endpoints(NodePort):
+  - 172.16.4.158:31441
+  - 172.16.4.155:31441
+```
+
+## tkctl get [component]
+
+This is a group of commands used to get the details of TiDB cluster componentes, the current TiDB cluster will be used if exists.
+
+Available components: `pd`, `tikv`, `tidb`, `volume`, `all`(query all components)
+
+| Flags | Shorthand | Description |
+| ----- | --------- | ----------- |
+| --tidb-cluster | -t | select the tidb cluster, default to current TiDB cluster |
+| --output | -o | output format, one of [default,json,yaml], the default format is `default` |
+
+Example:
+
+```
+$ tkctl get tikv
+NAME                  READY   STATUS    MEMORY          CPU   RESTARTS   AGE     NODE
+demo-cluster-tikv-0   2/2     Running   2098Mi/4196Mi         0          3m19s   172.16.4.155
```

Comment:
the `CPU`  column is empty now.

Reply:
In this example the CPU request and limit is not set.

Of course a `<none>/<none>` is more clear for user, I will open a new issue for this.

Reply:
```
[root@172.16.4.149 ~]# tkctl get tikv
NAME                     READY   STATUS    MEMORY          CPU   RESTARTS   AGE     NODE
cluster-restore-tikv-0   2/2     Running   2098Mi/8292Mi         0          5m22s   172.16.4.150
cluster-restore-tikv-1   2/2     Running   2098Mi/8292Mi         0          5m22s   172.16.4.154
cluster-restore-tikv-2   2/2     Running   2098Mi/8292Mi         0          5m22s   172.16.4.149
```

I try it on many clusters, but they are all empty.

Reply:
emmm... that's a bug

Reply:
I will amend the manual along with the fix #465